editor/editor_help.cpp

```diff
@@ -506,7 +506,19 @@ void EditorHelp::_update_doc() {
 	Set<String> skip_methods;
 	bool property_descr = false;
 
-	if (cd.properties.size()) {
+	bool has_properties = cd.properties.size() != 0;
+	if (cd.is_script_doc) {
+		has_properties = false;
+		for (int i = 0; i < cd.properties.size(); i++) {
+			if (cd.properties[i].name.begins_with("_") && cd.properties[i].description == String()) {
```

Comment:
Not too important, but I'd suggest using `.empty()` instead of ` == String()` to check for empty strings. (It's not done consistently in the codebase currently, we'll have to decide on a convention eventually.)

Reply:
done!